lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -203,6 +162,103 @@ sub _files_to_categories {
     return $ret;
 }
 
+my $rt_cpan_base = 'https://rt.cpan.org/Ticket/Display.html?id=';
```

Comment:
It kind of feels like it doesn't belong there, since it is display logic.  I wasn't sure really about where it should be, but ReleaseInfo would be the appropriate model if it was on that level.